misk/src/test/resources/misk/hibernate/docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+services:
+  vitess:
+    image: "vitess/base:latest"
+    command: "../../../bin/vttestserver -alsologtostderr -port 27000 -web_dir=web/vtctld/app -web_dir2=web/vtctld2/app -mysql_bind_host=0.0.0.0 -schema_dir=schema  -keyspaces=movies,actors --num_shards=2,1"
```

Comment:
You can make this a little more readable by splitting to multiple lines if you want. You can use an array of tokens
```
command: [
  "../../../bin/vttestserver",
  "-alsologtostderr",
  "-port", "27000",
   ...
]
```
or I believe the bar operator works here too:
```
command: |
  ../../../bin/vttestserver
  -alsologtostderr
  -port 27000
  ...
```

Reply:
Nice! I couldn't get the pipe thing to work though but the other one is nice enough. :-)